File: InventoryCommands/Domain/Events/StockAddedEvent.cs
using System;

namespace Domain.Events
{
	public class StockAddedEvent : IEvent
	{
		public Guid EventId { get; set; } = Guid.NewGuid();
		public Guid OrderId { get; set; }

		public Guid ProductId { get; set; }
		public int Amount { get; set; }

		public string EventName => "StockAdded";
	}
}

File: Shopping/ShoppingCommand/src/main/java/nl/avans/domain/models/events/product/StockAddedEvent.java
package nl.avans.domain.models.events.product;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;
import nl.avans.domain.models.events.Event;

import java.util.UUID;

@Data
public class StockAddedEvent implements Event {
    @JsonProperty
    private UUID customerId;

    @JsonProperty
    private UUID productId;

    @JsonProperty
    private int amount;
}
